pypuck/pypuck.py

```diff
@@ -139,7 +139,26 @@ def team_stats(start_season=None, end_season=None):
     Edmonton Oilers |  34  |   23   |  5 |   73   | ...
     ------------------------------------------------
     """
-    pass
+    
+    base_url = 'https://api.nhle.com/stats/rest/en/team/summary?'
+    arguments = 'cayenneExp=gameTypeId=2' +\
+          f' and seasonId<={end_season}' +\
+          f' and seasonId>={start_season}'
+    if str.isdecimal(start_season) == False:
+      print("Invalid Input. Enter start_season in Bi-Annual format. 'YYYYYYYY'")
+      return
+    if str.isdecimal(end_season) == False:
+      print("Invalid Input. Enter end_season in Bi-Annual format. 'YYYYYYYY'")
+      return
+    # print(base_url + arguments)
+    page = requests.get(base_url + arguments)
+    #print(page.url)
```

Comment:
No need to have this line remain in the code. Also, please add in test from helpers checking if status code is proper.

```suggestion

    # Check the response code is valid - i.e. the API didn't fail
    helpers.check_response_code(page.status_code)
```